src/Resolver.cpp

```diff
@@ -0,0 +1,126 @@
+/**
+ *
+ * @file Resolver.cpp DNS Resolver implementations
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#include "utilities/Resolver.hpp"
+
+std::vector<std::string>
+dunedaq::utilities::get_ips_from_hostname(std::string hostname, int port)
+{
+  std::vector<std::string> output;
+  std::string name = hostname;
+  std::string portstr = "";
+  if (port != 0)
+    portstr = std::to_string(port);
+
+  if (name.find(":") != std::string::npos) {
```

Comment:
Question for review: Should the port parsing still be here, or should the method rely on its parameters being filled out explicitly?